test/minikube/minikube_insights_test.go

```diff
@@ -54,28 +54,21 @@ func startAndScaleYCSB(t *testing.T, namespaceName string, options *helm.Options
 
 func checkMetricPresent(t *testing.T, namespace string, influxPodName string, influxDatabase string,
 	measurement string, database string, host string, metric string) bool {
-	queryString := fmt.Sprintf("select count(%s) from \"%s\" where host = '%s'", metric, measurement, host)
-	dbTagName := "db"
-	if influxDatabase == "nuodb_internal" {
-		// DB tag name is different in nuodb_internal database
-		dbTagName = "dbname"
-	}
-	if measurement != "nuodb_thread" {
-		// There is no db tag for nuodb_thread measurement
-		queryString = fmt.Sprintf("%s and %s = '%s'", queryString, dbTagName, database)
-	}
-	output, err := ExcuteInfluxDBQueryE(t, namespace, influxPodName, queryString, "-database", influxDatabase, "-format", "csv")
+	// queryString := fmt.Sprintf("select count(%s) from \"%s\" where host = '%s'", metric, measurement, host)
+	// dbTagName := "db"
+	query := fmt.Sprintf("from(bucket: \"%s\")\n |> range(start: -5m)\n |> filter(fn : (r) => r[\"_measurement\"] == \"%s\")\n |> filter(fn : (r) => r[\"_field\"] == \"%s\")\n |> filter(fn : (r) => r[\"host\"] == \"%s\") |> keep(columns: [\"_value\"]) |> count()", influxDatabase, measurement, metric, host)
+	output, err := ExcuteInfluxDBQueryE(t, namespace, influxPodName, query, "--raw")
 	if err != nil {
 		t.Logf("Unexpected error received from InfluxDB: %s", err)
 		return false
 	}
-	lines := strings.Split(output, "\n")
+	newOutput := strings.ReplaceAll(output, "\r\n", "\n")
+	lines := strings.Split(newOutput, "\n")
 	if len(lines) > 1 {
-		// The output format will be `name,time,count`
-		count, err := strconv.Atoi(strings.Split(lines[1], ",")[2])
+		count, err := strconv.ParseFloat(strings.Split(lines[4], ",")[3],64)
```

Comment:
You may want to run `go fmt ./...` so that it fixes the formatting for you. For example you should have spaces after the "," which separates the function arguments.